icepack/models/hybrid.py

```diff
@@ -228,9 +214,9 @@ def action(self, **kwargs):
 
         ds_w = ds_v(domain=mesh, subdomain_id=side_wall_ids)
         side_friction = self.side_friction(**kwargs) * ds_w
-        if get_mesh_dimensions(mesh) in ["xy", "xyz"]:
```

Comment:
I am confused on why we are dropping "xy" here, I could be missing something though.

Reply:
The hybrid model only works for "xz" and "xyz". We could add some checks that throw errors early on if you try to use a mesh with the wrong geometry. But if we're going to put effort into sanity-checking inputs there are other places that I think are much more prone to errors.

Reply:
Duh... ha, thanks.

Reply:
No worries! If it makes you feel any better, the reviewers for the icepack paper *hated* me calling this a "hybrid" model, so apparently it's confusing to everyone.